packages/editor/src/components/commands/index.js

```diff
@@ -98,8 +98,8 @@ function useEditorCommandLoader() {
 	commands.push( {
 		name: 'core/toggle-distraction-free',
 		label: isDistractionFree
-			? __( 'Exit Distraction Free' )
-			: __( 'Enter Distraction Free' ),
+			? __( 'Exit Distraction free' )
+			: __( 'Enter Distraction free' ),
```

Comment:
Should this include the word "mode"  like in the actions and for the Spotlight mode?

Reply:
Not sure. In the UI it's called 'Distraction free':

![Screenshot 2024-10-29 at 10 33 23](https://github.com/user-attachments/assets/6c8fcb72-7e45-4147-a995-122a6f340bc8)

Reply:
I see.
Then is it a problem that the actions use "mode" ?